src/pkg/bundle/bundle.go

```diff
@@ -221,6 +257,16 @@ func CreateAndPublish(remoteDst *oci.OrasRemote, bundle *types.UDSBundle, signat
 		// ensure media type is a Zarf blob and append to bundle root manifest
 		zarfManifestDesc.MediaType = oci.ZarfLayerMediaTypeBlob
 		message.Debugf("Pushed %s sub-manifest into %s: %s", url, dstRef, message.JSONValue(zarfManifestDesc))
+
+		// add package name annotations to zarf manifest
+		zarfYamlPackage, err := remoteBundler.RemoteSrc.FetchZarfYAML()
```

Comment:
Based on the var name, I'm not sure what `zarfYamlPackage` is. Is it the `zarf.yaml`?

Reply:
yup, updated to "zarfYamlFile"